src/components/Tool/GoogleImport/GoogleImport.test.tsx

```diff
@@ -0,0 +1,368 @@
+import { ApolloCache } from '@apollo/client';
+import { ThemeProvider } from '@mui/material/styles';
+import { render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ApolloErgonoMockMap } from 'graphql-ergonomock';
+import { MockLinkCallHandler } from 'graphql-ergonomock/dist/apollo/MockLink';
+import { getSession } from 'next-auth/react';
+import { SnackbarProvider } from 'notistack';
+import TestRouter from '__tests__/util/TestRouter';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { fetchAllData } from 'src/lib/deserializeJsonApi';
+import theme from 'src/theme';
+import GoogleImport from './GoogleImport';
+import { mockGoogleContactGroupsResponse } from './GoogleImportMocks';
+import { GoogleContactGroupsQuery } from './googleContactGroups.generated';
+
+const mockEnqueue = jest.fn();
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+jest.mock('src/lib/deserializeJsonApi');
+jest.mock('next-auth/react');
```

Comment:
`getSession` from `next-auth/react` is already mocked for all tests in `__tests__/util/setup.ts`, so you don't need to mock it again.